content/en/blog/2021-12-23-data-curation-baleen.md

```diff
@@ -0,0 +1,57 @@
+
+> GPT-3, like all large language models trained on internet corpora, will generate stereotyped or prejudiced content[^9]
+
+The goal of training GPT-3 on a large, general dataset may have been to imbue the model with general intelligence, but unfortunately its creators also unknowingly fed toxic elements into the machine learning model. As a result, GPT-3 is ultimately limited by the data used to teach it to "speak", much of it rife with racism, sexism, and religious prejudice. This is one of the key [pitfalls of megamodels](https://rotational.io/blog/a-parrot-trainer-eats-crow/) trained on uncurated text.
+
+Are models better when trained on more specific data? Github's Copilot may be an example of AI that is trained on custom dataset. Copilot (run by OpenAI Codex) is designed to help developers generate code faster. It works by "understanding" the previous lines of code and suggesting individual lines and whole functions instantly[^10], and is trained on code respositories on Github.[^11] In order to properly utilize NLP, AI and deep learning, we need to produce quality models that contain quality data.
+
+## Getting Better Data for AI/ML
+
+So how can you create your own curate data to ensure your machine learning models have the best quality input? At Rotational, our emphasis is to approach NLP (and other ML problems) by first generating and storing custom datasets to allow for better machine learning models. This is where projects like Baleen can fill in the gap.
+
+[Baleen](https://github.com/rotationalio/baleen) is an example of an automated ingestion service for RSS feeds to construct a corpus for NLP research. Written in Golang, the ingestion system fetches RSS feeds and stores raw data into S3. Baleen also collects data quality measurements with language statistics, such as total words, vocabulary (unique words), hapaxes (words that only occur once through the corpus), rate of corpus growth, number of entities, etc.[^12] Users can specify their own [custom list of RSS feeds](https://github.com/rotationalio/baleen/tree/develop/fixtures) to Baleen. Note that because  publishers and authors of the original articles hold the copyright to their works, data collected via Baleen should not be used for commercial purpose, only for educational purposes. Nonetheless, Baleen presents an example that others can use as a model for data collection and curation.
+
+
+***
+
+Photo by [Art of Backpacking](https://flic.kr/p/8GAVjS) on [Flickr Commons](https://flic.kr/p/8GAVjS)
+
+---
+[^1]: https://builtin.com/artificial-intelligence/ai-companies-roundup
+[^2]: https://www.urbint.com/about
+[^3]: https://www.datarobot.com/abo
```

Comment:
This link did not work for me; could you please correct it and also double check the other links to make sure all of them are working?